sunpy/net/helio/hec.py

```diff
@@ -44,7 +50,62 @@ def votable_handler(xml_table):
     return votable
 
 
-class HECClient:
+class HECResponse(BaseQueryResponse):
```

Comment:
This modeled off the JSOC response?

Reply:
`HECResponse` is almost `JSOCResponse`, just without `query_atts` and `errors`